src/Notes/MerchantEmailNotifications/NotificationEmail.php

```diff
@@ -135,7 +135,7 @@ public function get_content_html() {
 		return wc_get_template_html(
 			$this->get_template_filename( 'html' ),
 			array(
-				'email_actions'           => $this->format_string( $this->get_actions() ),
+				'email_actions'           => $this->get_actions(),
```

Comment:
cc'ing @ilyasfoo given you [added the `format_string`](https://github.com/woocommerce/woocommerce-admin/pull/6617) last, in the PHP8 tests it error'ed as `actions` was an object trying to convert it to a string. From looking at the template I assume it should stay an object?

Reply:
You're right, thanks for catching this! I missed to test actions with `format_string`